### DISCUSSION TITLE
SHA256 sum of cask without installation

### DISCUSSION THREAD

**Discussion by guadagn0** [Homebrew/discussions#3210] [Everyday usage] [answered]
Is there a way to obtain the SHA256 sum of a given homebrew cask, without installing such cask?

I'm asking because I installed a specific application (https://www.irradiatedsoftware.com/sizeup/) directly from their website, which however doesn't offer a way to verify the integrity of the download. Only afterwards I realized that the same application was available as homebrew cask.

Thank you. (Homebrew is amazing.)

**Comment by SMillerDev** [comment, marked as answer]
There is `brew fetch sizeup` to download and print the shasum, `brew info sizeup --json=v2` or `brew cat sizeup` and the info data is also available at https://formulae.brew.sh/api/cask/sizeup.json.
